atom/common/chrome_version.h

```diff
@@ -8,7 +8,7 @@
 #ifndef ATOM_COMMON_CHROME_VERSION_H_
 #define ATOM_COMMON_CHROME_VERSION_H_
 
-#define CHROME_VERSION_STRING "66.0.3359.181"
+#define CHROME_VERSION_STRING "67.0.3396.99"
```

Comment:
would be sweet to generate this file.

Reply:
+1
Maybe we even could even be getting this version string from Chromium itself?